### ISSUE TITLE
Create p5 sketches for figures I.2, I.3, I.5, I.6

### ISSUE THREAD

**Issue by shiffman** [nature-of-code/noc-book-2#104]
@zee-em do you think it makes sense to draw these or should I just regenerate them with p5?

<img width="1079" alt="Screen Shot 2022-10-11 at 1 47 04 PM" src="https://user-images.githubusercontent.com/191758/195163146-886b2123-d4fb-4f76-8925-d78cea2152e6.png">

**Comment by zee-em** [comment]
Is there a reason they were generated as sketches, like would people be creating that output themselves to test in p5, and this is an example of what to expect? 

If not, I would say let's do them as illustrations for consistency's sake. 

BTW it seems like there's an opportunity to add a fun monkey illustration in this section, with tall and short monkeys... Maybe a New York City street scene made up of monkeys of varying heights... just a thought...

**Comment by shiffman** [comment]
It's nice to have the accompanying code as an extra resource for these illustrations (and that's probably why I did them this way in the first place) but there's really no reason to have them be screenshots since they are static diagrams. (In a future iteration of the book I could imagine an interactive diagram where you get to play around with sliders.) I think I would opt for drawn (and I love the idea of monkeys! Or cats! Or chickens! 😂)

There's another similar spot later in the intro that looks like this:

<img width="806" alt="Screen Shot 2022-10-11 at 8 43 30 PM" src="https://user-images.githubusercontent.com/191758/195223496-73aa088b-66ef-4058-881d-b35af74b9981.png">

I do mention the accompanying code, but these could be drawn as well? Maybe you would even want to trace them? Or is that silly?

@jasongao97 do we have a way of embedding a p5 sketch with link to the web editor if it's not an example? B/c that's another option that would be a reason to have them be p5 canvases rather than illustrations?

**Comment by jasongao97** [comment]
Sure! Here is one already. (2D Perlin noise) Embedded sketches are now represented just by a bookmark in notion so they can be anywhere.

<img width="967" alt="image" src="https://user-images.githubusercontent.com/6762203/195231016-8a771003-f50e-4bc6-b968-bcee0e77fae7.png">

**Comment by shiffman** [comment]
Ah, thanks @jasongao97! @zee-em let's hold off on illustrating these and I'll first try embedding them as p5 sketches. But I do like the idea of the monkey illustration to help explain so feel free to try out ideas there!

@jasongao97 I'm loathe to add features which in turn add complexity but it does occur to me that in the `reset` `pause` bar we could consider a mechanism for adding a slider that controls a variable from the sketch? Maybe file this as an idea for later?

Let's leave this open and I'll re-title to indicate that i need to add the p5 sketches as bookmarks.

**Comment by zee-em** [comment]
got it- no illos for these two :)

**Comment by shiffman** [comment]
As @jasongao97 and I just discussed in our meeting, the only additional feature we need is adding the support for a caption #165. I'm working on these today, @zee-em I combined I.2 and I.3 into one so I ended up renumbering the illustrations in this chapter in case that affects your file management. The new numbers are updated in Notion.

**Comment by shiffman** [comment]
These are all now p5 sketches in the Introduction. I also replaced a screenshot for Exercise I.10 with a p5 sketch. The only thing that is a little too bad about this approach is I can't see the sketch itself in notion, but this is a tiny thing as I always click on it and run the sketch.

For each of these I've uploaded `screenshot.png` to the p5 sketch for the print version.

![Screen Shot 2022-11-16 at 2 34 54 PM](https://user-images.githubusercontent.com/191758/202277113-f8b105ed-62d0-4139-94c9-559d38ce3c6e.png)

@jasongao97 I'm re-assigning this issue to you. Once #165 is resolved and the intro chapter builds correctly, it can be closed.
